File: Level1/두 개 뽑아서 더하기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> numbers) {
    vector<int> answer;
    bool check[200] = {false, };
    
    for(int i = 0; i < numbers.size(); i++)
    {
        for(int j = i + 1; j < numbers.size(); j++)
        {
            int sum = numbers[i] + numbers[j];
            
            if(check[sum] == false)
            {
                answer.push_back(sum);
                check[sum] = true;
            }
        }
    }
    
    sort(answer.begin(), answer.end());
    return answer;
}

File: Level1/두 개 뽑아서 더하기.js
function solution(numbers) {
  let answer = [];

  numbers.filter((val, index) => {
    for (let i = index + 1; i < numbers.length; i++) {
      answer.push(val + numbers[i]);
    }
  });

  answer = Array.from(new Set(answer)).sort((a, b) => {
    return a - b;
  });

  return answer;
}
